contracts/pair/src/contract.rs

```diff
@@ -80,6 +80,7 @@ pub fn instantiate(
     }
 
     let pair_info: &PairInfo = &PairInfo {
+        owner: info.sender,
```

Comment:
`info.sender` is the Factory contract because Pair instantiate endpoint is called in `create_pair` in Factory contract.
So in Pair contract `try_update_config` endpoint should be called in the Factory contract, in another word, you can't call this endpoint straight.

Please refer this code.
https://github.com/AngelProtocolFinance/angelprotocol-smart-contracts/pull/195/commits/10c6dad6d36cf13312d00cd384f3a76490cbcd00#diff-88ae66b470ef0d7ffabe008374c51826ed09172c4198fc0e3af7f58bf6a6ba3fR103

Reply:
This was already implemented in [521d6557](https://github.com/AngelProtocolFinance/astroport-lbport/pull/1/commits/521d6557c171d1ad9cb2d37b909689684750fcef).